spec/truffle/launcher_spec.rb

```diff
@@ -128,4 +128,16 @@ def should_print_full_java_command(cmd)
     out.lines[1].should == "14\n"
   end
 
+  it "doesn't include the polyglot.ruby prefix in the list of options" do
+    out = `#{RbConfig.ruby} -Xoptions`
+    $?.success?.should == true
+    out.should_not include("polyglot.ruby")
```

Comment:
Yes because it was actually printing `ruby.` rather than `polyglot.ruby.` wasn't it.